keras_cv/training/contrastive/contrastive_trainer_test.py

```diff
@@ -24,6 +25,7 @@
 from keras_cv.training import ContrastiveTrainer
 
 
```

Comment:
Should we leave a comment here about why these are XL?
Maybe something like
"These tests are expensive and our contrastive training API is not yet released"?